scheduler/config-k8s.edn

```diff
@@ -95,7 +95,9 @@
                         :resource-requirements {:cpu-request 0.1
                                                 :cpu-limit 0.1
                                                 :memory-request 128.0
-                                                :memory-limit 128.0}}}
+                                                :memory-limit 128.0}}
+              :disk-type-node-label-name [{:pool-regex "^k8s-alpha$" 
+                                           :disk-node-label "cloud.google.com/gke-boot-disk"}]}
```

Comment:
yeah, that makes sense. I think this was first a separate section because we assumed that the node-label would be the same for different pool-regex's, but I have now added it to the disk section